### PR TITLE
PanelSprite JavaScript function annotations

### PR DESCRIPTION
**Pull request by Wend1go**

### CODE REVIEW

File: Extensions/PanelSpriteObject/panelspriteruntimeobject.js [hunk 1]
```diff
@@ -59,38 +59,71 @@ gdjs.PanelSpriteRuntimeObject.prototype.extraInitializationFromInitialInstance =
...
+ */
 gdjs.PanelSpriteRuntimeObject.prototype.setY = function(y) {
     gdjs.RuntimeObject.prototype.setY.call(this, y);
     this._renderer.updatePosition();
 };
 
+/**
+ * Set the texture of the panel sprite.
+ * @param {string} textureName The name of the texture.
+ * @param {RuntimeScene} runtimeScene The scene the object lives in.
```
Comment (4ian) [inline Extensions/PanelSpriteObject/panelspriteruntimeobject.js]: I think you need to specify `{gdjs.RuntimeScene}`?

Comment (Wend1go) [reply Extensions/PanelSpriteObject/panelspriteruntimeobject.js]: What do you mean?

Comment (4ian) [reply Extensions/PanelSpriteObject/panelspriteruntimeobject.js]: Yes I think it's required to have the "fully qualified" name: `gdjs.RuntimeScene` and not simply `RuntimeScene` (which would be a global class, while `RuntimScene` leaves inside `gdjs` object).

**Comment by 4ian** [comment]
Great 👍

**Pull request merged**